frontend/src/components/user-profile/styles.module.scss

```diff
@@ -52,6 +52,11 @@
         .headerLogo {
           margin-right: 5px;
         }
+
+        .emailLink {
+          text-decoration: none;
+          color: #4c4c4d;
```

Comment:
![image](https://user-images.githubusercontent.com/42296182/129471639-4f57021e-03e0-4233-ab17-8a8bfb7d28a2.png)
on design we have this color 🤔